packages/nouns-contracts/contracts/governance/NounsDAOInterfaces.sol

```diff
@@ -641,6 +641,6 @@ interface NounsDAOV3Types {
     }
 }
 
-contract NounsDAOStorageV3 is NounsDAOV3Types {
+contract NounsDAOStorage is NounsDAOV3Types {
     StorageV3 ds;
```

Comment:
@eladmallel this part feels a bit off to me
still referencing V3 is many names
also, we did change the storage layout, so maybe it makes sense to change the name?

Reply:
updated these names as well as the names of the DAO libraries